### DISCUSSION TITLE
How to trigger automatic retry earlier

### DISCUSSION THREAD

**Discussion by snawaz** [OptimalBits/bull#2350] [General]
I'm using automatic retries enabled by backoff option. Let's assume that the retry delay is `1 hour`. That is the default. In my case, the retries are there because the jobs could wait for certain information, like some status update on something else. Now consider this scenario that I get the status update for a particular job using webhooks and at that point I want this job to be retried immediately. Is there a way to do that? Let's assume that in the webhooks handlers, I've access to the `jobId` and `queue` (I could change my code to have these info).

**Comment by manast** [comment]
Then you just use "promote" https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#jobpromote
